packages/server/src/integrations/tests/utils/postgres.ts

```diff
@@ -3,45 +3,45 @@ import { GenericContainer, Wait, StartedTestContainer } from "testcontainers"
 
 let container: StartedTestContainer | undefined
 
-export async function getDsConfig(): Promise<Datasource> {
-  try {
-    if (!container) {
-      container = await new GenericContainer("postgres:16.1-bullseye")
-        .withExposedPorts(5432)
-        .withEnvironment({ POSTGRES_PASSWORD: "password" })
-        .withWaitStrategy(
-          Wait.forLogMessage(
-            "database system is ready to accept connections",
-            2
-          )
-        )
-        .start()
-    }
-    const host = container.getHost()
-    const port = container.getMappedPort(5432)
+export async function start(): Promise<StartedTestContainer> {
+  if (!container) {
+    container = await new GenericContainer("postgres:16.1-bullseye")
+      .withExposedPorts(5432)
+      .withEnvironment({ POSTGRES_PASSWORD: "password" })
+      .withWaitStrategy(
+        Wait.forSuccessfulCommand(
+          "pg_isready -h localhost -p 5432"
+        ).withStartupTimeout(10000)
```

Comment:
Do we need this timeout? Should we not rely on the jest timeout?

Reply:
I'm not sure, I copied what existed already. Happy to remove and see what happens.